.expeditor/config.yml

```diff
@@ -61,9 +41,22 @@ staging_areas:
 
 subscriptions:
   # These actions are taken, in order they are specified
+  - workload: pull_request_opened:{{agent_id}}:*
+    actions:
+      - post_github_comment:.expeditor/templates/welcome.mustache
+
   - workload: pull_request_merged:{{agent_id}}:*
     actions:
-      - trigger_pipeline:release_habitat
+      - built_in:bump_version:
+          ignore_labels:
+          - 'Expeditor: Skip Version Bump'
+          - 'Expeditor: Skip All'
+      - built_in:update_changelog:
+          ignore_labels:
+          - 'Expeditor: Exclude from Changelog'
+          - 'Expeditor: Skip All'
+      - trigger_pipeline:release_habitat:
```

Comment:
Do you want to put a filter on this action to only trigger if you bump the version? 

```
- trigger_pipeline:release_habitat:
    only_if: built_in:bump_version
    post_commit: true
```